tests/bluetooth/bsim_bt/bsim_test_adv_resume/prj.conf

```diff
@@ -0,0 +1,21 @@
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
+
+CONFIG_ASSERT=y
+CONFIG_BT_TESTING=y
+CONFIG_BT_DEBUG_LOG=y
+
+CONFIG_BT_EXT_ADV=n
+CONFIG_BT_PRIVACY=n
+CONFIG_BT_SCAN_WITH_IDENTITY=n
+CONFIG_BT_LOG_SNIFFER_INFO=y
+
+CONFIG_BT_AUTO_PHY_UPDATE=n
+CONFIG_BT_AUTO_DATA_LEN_UPDATE=n
+CONFIG_BT_GAP_AUTO_UPDATE_CONN_PARAMS=n
+
+# Set to 3 to see the problem go away
+CONFIG_BT_MAX_CONN=2
+CONFIG_BT_MAX_PAIRED=2
+CONFIG_BT_ID_MAX=2
```

Comment:
Is the comment still valid? What problem is it referring to?

Reply:
It is now invalid. The first commit introduces a failing test (ie, the "problem"). If this is set to 3, then the test passes.
I'll remove it altogether.